app/styles/new-signup.css

```diff
@@ -0,0 +1,92 @@
+:root {
+    --container-color: #666;
+    --user-details-background: #f0f2f5;
+    --heading-color: #000000f2;
+    --input-border: #a9a9a9;
+    --button-background: #008000;
+    --button-color: #ffffff;
+    --button-background-disabled: #ccc;
+    --error: #ff0000;
```

Comment:
Make the variable name as the color name, that way it can be used for any background, text color, etc.